src/components/Composer/index.js

```diff
@@ -134,8 +136,21 @@ class Composer extends React.Component {
                 end: initialValue.length,
             },
         };
-        this.dragNDropListener = this.dragNDropListener.bind(this);
         this.paste = this.paste.bind(this);
+
+        this.dropZone = document.getElementById(CONST.REPORT.DROP_NATIVE_ID);
+        this.dropZoneRect = this.calculateDropZoneClientReact();
+        this.dragNDropListener = this.dragNDropListener.bind(this);
+        this.dragNDropWindowResizeListener = this.dragNDropWindowResizeListener.bind(this);
```

Comment:
Maybe we should throttle this window resize listener?

```suggestion
        this.dragNDropWindowResizeListener = _.throttle(this.dragNDropWindowResizeListener.bind(this), 100);
```

Reply:
Agree